CodeEditModules/Package.swift

```diff
@@ -127,6 +131,11 @@ let package = Package(
                 "WorkspaceClient"
             ],
             path: "Modules/Search/src"
+        ),
+        .target(
+            name: "GitClone",
+//            dependencies: ["GitClient"],
```

Comment:
You probably need to uncomment this so tests pass.
![Screen Shot 2022-03-25 at 03 21 38](https://user-images.githubusercontent.com/9460130/160041937-210e940e-5ce0-4bdb-9f73-19477dc751c6.png)